### PR TITLE
New example has been added for demonstrate large file upload

### PR DESCRIPTION
**Pull request by ethrbh**
hello,

I made a new example for demonstrate how large size file can be uploaded to the web server. Please review it.

thanks,
/Robi

### CODE REVIEW

File: examples/upload_large_file/src/upload_large_file_handler.erl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	{Data, Req2} = acc_multipart(Req, []),
+		
+	[{Headers, Body}] = Data,
+	
+	{_, << FileData/binary >>} = lists:keyfind(<<"content-disposition">>, 1, Headers),
+	
+	%% Note: The FileData looks like this: "form-data; name=\"inputfile\"; filename=\"test_upload.txt\""
+	[_FormData, _Name, FileNameTmp] = string:tokens(erlang:binary_to_list(FileData), ";"),
+	[_, FileNameTmp2] = string:tokens(FileNameTmp, "="),
+	FileName = string:strip(FileNameTmp2, both, $"),
```
Comment (essen) [inline examples/upload_large_file/src/upload_large_file_handler.erl]: We have `cow_multipart` for that, though currently undocumented.

**Comment by ethrbh** [comment]
hello Essen,

I have made the changes for using cow_multipart:parse_content_disposition/1. Please check out.

thanks,
/Robi

**Comment by ethrbh** [comment]
hello Essen,

Did you have time to check the changes I did? Do I need change more, or the new example looks good enough?

thanks,
/Robi

**Comment by essen** [comment]
Finishing the manual before looking at other tickets.

**Comment by essen** [comment]
I would rather just modify the existing upload example to accept any size.

**Comment by ethrbh** [comment]
hello Essen,

To improve the existing example also looks good. Than, you can close this ticket, I guess.

br,
/Robi

**Comment by essen** [comment]
Please do send a new PR with the existing example modified though. I am currently reviewing all tickets/PRs. Thanks!

**Pull request closed without merging**